tck/README.adoc

```diff
@@ -28,65 +28,87 @@ must be executed to validate compliance.
 The optional TCK currently validates the following non-mandatory features:
 
 - REST stat metrics
+- Base metrics
 
 == Dependencies
 
 To enable the tests in your Arquillian based test project you need to add the following dependencies:
 
 [source, xml]
 ----
+
+<properties>
+    <version.microprofile.metrics>5.0.0</version.microprofile.metrics>
+    <version.jakartaee.coreprofile>10.0.0</version.jakartaee.coreprofile>
+    <version.junit>4.13.2</version.junit>
+    <version.arquillian>1.7.0.Alpha12</version.arquillian>
+    <version.hamcrest>1.3</version.hamcrest>
+</properties>
+
 <dependencies>
     <dependency>
         <groupId>org.eclipse.microprofile.metrics</groupId>
         <artifactId>microprofile-metrics-api</artifactId>
-        <version>3.0.0</version>
+        <version>${version.microprofile.metrics}</version>
     </dependency>
 
     <dependency>
         <groupId>org.eclipse.microprofile.metrics</groupId>
         <artifactId>microprofile-metrics-rest-tck</artifactId>
-        <version>3.1</version>
+        <version>${version.microprofile.metrics}</version>
     </dependency>
 
     <dependency>
         <groupId>org.eclipse.microprofile.metrics</groupId>
         <artifactId>microprofile-metrics-api-tck</artifactId>
-        <version>3.1</version>
+        <version>${version.microprofile.metrics}</version>
     </dependency>
 
+    <!-- version.jakartaee.coreprofile should correspond to the value 
+    specified in the MicroProfile Parent pom.xml used by the specified 
+    version of MicroProfile Metrics -->
     <dependency>
-        <groupId>jakarta.enterprise</groupId>
-        <artifactId>jakarta.enterprise.cdi-api</artifactId>
-        <version>3.0.0</version>
+        <groupId>jakarta.platform</groupId>
+        <artifactId>jakarta.jakartaee-core-api</artifactId>
+        <version>${version.jakartaee.coreprofile}</version>
         <scope>provided</scope>
     </dependency>
 
+    <!-- version.junit should correspond to the value specified in the
+     MicroProfile Parent TCK-bom pom.xml used by the specified version of
+     MicroProfile Metrics -->
     <dependency>
         <groupId>junit</groupId>
         <artifactId>junit</artifactId>
-        <version>4.12</version>
+        <version>${version.junit}</version>
         <scope>test</scope>
     </dependency>
 
+    <!-- version.hamcrest should correspond to the value specified in the
+     MicroProfile Parent TCK-bom pom.xml used by the specified version of
+     MicroProfile Metrics -->
     <dependency>
         <groupId>org.hamcrest</groupId>
         <artifactId>hamcrest-all</artifactId>
-        <version>1.3</version>
+        <version>${version.hamcrest}</version>
```

Comment:
I think this dependency was defined in the microprofile-parent as well.

Reply:
This is for the README for tck runner